index.html

```diff
@@ -526,16 +538,16 @@ <h2>eBraille file set</h2>
 			<section id="fileset-intro" class="informative">
 				<h3>Introduction</h3>
 
-				<p>The [=eBraille file set=] is like a physical manifestation of the <a
+				<p>The [=eBraille file set=] is essentially an <a
 						data-cite="epub3#sec-container-abstract">OCF abstract container</a> [[epub3]]. EPUB 3 only
 					defines its file set in the abstract because those files are expected to be zipped in the <a
 						data-cite="epub3#dfn-epub-container">EPUB container</a> [[epub3]]; the standard is not concerned
 					with the physical files before they are zipped or after they are unzipped. As a ZIP file does not
 					have a true file system within it, the rules for file naming and placement can only be defined
 					virtually.</p>
 
-				<p>eBraille moves the rules for the OCF abstract container to the physical file set that exists before
-					and after packaging in the EPUB container. This allows an [=eBraille publication=] to be independent
+				<p>Unlike EPUB, eBraille allows materialization of the OCF abstract container both as a physical file set
```

Comment:
Minor nit, but "materialization" has a sudden appearance ring to it (often used for ghosts appearing). Maybe these uses should be "manifestation" to match the EPUB spec.

Reply:
sure, "manifestation" sounds good!